__tests__/utils.test.js

```diff
@@ -4,6 +4,48 @@ const {
   formatComments,
 } = require("../db/seeds/utils");
 
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data/index.js");
+const { forEach } = require("../db/data/test-data/articles.js");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("Northcoders News API", () => {
+  describe("api/", () => {
+    describe("Status health check", () => {
+      test("200 - respond with the 200 status code", () => {
+        return request(app).get("/api/healthcheck").expect(200);
+      });
+    });
+
+    describe("/api/topics/", () => {
+      test("responds with an array of topic objects", () => {
+        return request(app)
+          .get("/api/topics")
+          .expect(200)
+          .then((response) => {
+            return response.body;
+          })
```

Comment:
You don't need to have two `then` blocks here, the return of `response.body` isn't resolving a _Promise_ so it can all be done inside one `then`